src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -135,14 +137,14 @@
                     LOGGER.info("Security analysis result '{}' available for study '{}' and user '{}'",
                             resultUuid, receiverObj.getStudyName(), receiverObj.getUserId());
 
+                    studyRepository.updateSecurityAnalysisResultUuid(receiverObj.getStudyName(), receiverObj.getUserId(), resultUuid);
```

Comment:
maybe move this in Mono.fromCallable below with the studyChanged to make the code more simple ? In a later PR we can cleanup this code which is more complicated than it needs to be

Reply:
i don't find a need for that !